### PR TITLE
Added connection string information to quick start document

### PR DESCRIPTION
**Pull request by zulkiflifauzi**
## Summary
Added how to override connection string information

### CODE REVIEW

File: docs/home/start.md [hunk 1]
```diff
@@ -58,7 +58,12 @@ You will need to build the API, Engine, and UI (either CLI or Web) to run `OpenC
 
 > Note that running this script will run all 4 of the components. If you want to skip CLI or Web, you can add the `-noCli` or `-noWeb` option. For example if you want to go all CLI, use the following command instead: `.\builds\build-all.ps1 -noWeb`
 
-The script will ask you to input the connection string for the API. It will also automatically open new PowerShell windows for other components.
+The script will ask you to input the connection string for the API in the OpenCatapult API PowerShell window. It will also automatically open new PowerShell windows for other components. The default connection string is:
+
+```powershell
+"Server=localhost;Database=OpenCatapult;User ID=opencatapult;Password=opencatapult;"
```
Comment (BrainBalaka) [inline docs/home/start.md]: I think the default connection string when we first clone the opencatapult is: `Server=localhost;Database=polyrific.catapult.db;User ID=sa;Password=samprod;`

**Comment by frandi** [comment]
Please also improve the PR title with the more descriptive one, thanks!

File: docs/home/start.md (hunk 1 above)
Comment (zulkiflifauzi) [reply docs/home/start.md]: Thank you for the review!

**Comment by zulkiflifauzi** [comment]
hi @BrainBalaka could you please review this changes again? thanks!

**Pull request merged**